shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/domain/usecase/LoginUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.mark.kmmic.domain.usecase
+
+import co.nimblehq.mark.kmmic.domain.model.AuthToken
+import co.nimblehq.mark.kmmic.domain.repository.AuthRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.onEach
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+interface LoginUseCase {
+    fun call(email: String, password: String): Flow<AuthToken>
```

Comment:
Please use `invoke` (same with `callAsFunction` in Swift)

Reply:
@suho updated at [b5c86d3](https://github.com/markgravity/kmm-ic/pull/67/commits/b5c86d34cf43b9e1cac1bdf1dc2ae0f36ddf159a)